format/FormatMultiImage.py

```diff
@@ -228,29 +228,30 @@ def get_imageset(
                 indices=single_file_indices,
             )
 
+            if single_file_indices is None:
```

Comment:
Is it valid to pass `single_file_indices=[]` to this function? Or if not, could this test be simply `if single_file_indices:`? And if an empty list _is_ valid, is it tested?
```suggestion
            if single_file_indices is None:
```

Reply:
I can't see why this would be passed an empty list of single_file_indices, but if someone did so, I imagine they would expect to get an empty image set. I would rather test `is None`. HOWEVER. It seems if we call `ImageSet([...], indices=[])`, that empty list is treated the same as `None`. To avoid inconsistency I would propose just ensuring that `single_file_indices` is either `None` or a non-empty list.

Reply:
Please see https://github.com/cctbx/dxtbx/pull/210/commits/d095865e335d37645f4bdc4bfbcd0d16923c812b and let me know what you think